Homework1/Homework1/Program.cs

```diff
@@ -0,0 +1,449 @@
+﻿namespace Homework1
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            ManagementRestaurant managementRestaurant = new ManagementRestaurant();
+
+            while (true)
+            {
+                Console.WriteLine("Choose what you want (1 - add, 2 - watch, 3 - create order, 4 - stop app)");
+                string? answerUserMethods = Console.ReadLine();
```

Comment:
Why did you add '?' symbol at the end of string class name?

Reply:
Because without '?' he swears that there may be a null value